### ISSUE TITLE
'heeft gegevensgroeptype' foutief gedefinieerd op 'gegevensgroeptype'

### ISSUE THREAD

**Issue by melsk-r** [Geonovum/MIM-Werkomgeving#492]
In paragraaf '3.2.1.4 «Gegevensgroeptype»' van de consultatieversie 1.2 van het MIM is het metagegeven 'heeft gegevensgroeptype' opgenomen in de tabel. Dat lijkt me een foutje wat volgens mij wordt bevestigd doordat in paragraaf '2.4.4.3 Metagegeven: heeft gegevensgroeptype' wordt aangegeven dat dit metagegeven alleen van toepassing is op «Gegevensgroep».

**Comment by PalmJanssen** [comment]
Klopt, dit is een fout in die tabel 3.2.1.4. Een `Gegevensgroeptype` kan geen `heeft gegevensgroeptype` hebben. Wel een `heeft gegevensgroep`. Deze fout is in de rest van het doc verder nergens toegepast (niet in UML en niet in LD implementatie). Kan dus als bug zonder implicatie opgelost worden.

Dit is nog gecorrigeerd in de fase van `versie ter vaststelling` naar `vastgestelde versie`

**Comment by PalmJanssen** [comment]
is opgelost
